src/main/fc/init.c

```diff
@@ -295,6 +297,12 @@ void init(void)
     }
 #endif
 
+    // Configure MCO output after config is stable
+
+#ifdef USE_MCO
+    mcoInit(mcoConfig());
```

Comment:
@RicoKirstein: Just came across this as part of another pull request: I think this should ideally be moved to after the `OverclockRebootIfNecessary()` call - no point running `mcoInit()` twice in the case that we need to reboot for overclocking.

Reply:
@mikeller you might be right. It doesn't hurt to call it twice but it also should work if called after OC initialization. I never tried OC on my board since it's already getting really hot while not flying. Should I create a new pr, or will you update it with another pr?

Reply:
@RicoKirstein: I don't have hardware to test this on easily, so it's probably best if you make the change, then confirm it's working as it should, and open a pull request. Thanks.